src/Sulu/Bundle/MediaBundle/Resources/public/js/components/media-selection/main.js

```diff
@@ -76,32 +77,21 @@ define(['sulumedia/collection/collections', 'sulumedia/model/collection'], funct
         },
 
         templates = {
-            addTab: function(options, header) {
+            mediaSelection: function(options) {
                 return [
-                    '<div id="', options.ids.chooseTab, '">',
-                    '   <div class="heading">',
-                    '       <h3>', header, '</h3>',
+                    '<div class="media-selection-overlay">',
+                    '   <div class="media-selection-overlay-navigation-container pull-left"></div>',
+                    '   <div class="media-selection-overlay-content">',
+                            '<div class="fa-times media-selection-overlay-close"></div>',
```

Comment:
Put the spaces into the string, so that reformatting the code doesn't break indentation.